pep-0503.txt

```diff
@@ -96,6 +96,13 @@ In addition to the above, the following constraints are placed on the API:
   a value of either ``true`` or ``false`` to indicate whether or not there is a
   GPG signature. Repositories that do this **SHOULD** include it on every link.
 
+* A repository **MAY** include a ``data-requires-python`` attribute on a file
+  link. This exposes the *Requires-Python* metadata field, specified in PEP 345,
+  for the corresponding release. Where this is present, installer tools
+  **SHOULD** ignore the download when installing to a Python version that
+  doesn't satisfy the requirement. For example::
+
+      <a href="..." data-requires-python=">=3">...</a>
```

Comment:
`<` needs encoding in every flavour of HTML/SGML/XML/etc.
`>` needs to be encoded for some of them (sorry I dont recall the specifics offhand), but not all of them, so generally people and tools encode it all the time, to be safe and future proof in case the DOCTYPE of the document is changed in the future, or a browser implicitly detects it as a different DOCTYPE than the writer intended.